arangod/Aql/OptimizerRules.cpp

```diff
@@ -1819,6 +1819,8 @@ void arangodb::aql::moveCalculationsDownRule(
         continue;
       }
       variable = nn->outVariable();
+    } else {
+      TRI_ASSERT(false);
```

Comment:
just replace `else if` with `else` and write assert in it.
Can be production assert if you're not sure